### ISSUE TITLE
Unable to run the executable

### ISSUE THREAD

**Issue by iShafayet** [Theo47/depressurizer#77]
I'm running the latest release ( https://github.com/Theo47/depressurizer/releases/tag/v0.7.1.4 ) on Windows 10 64bit.

I simply downloaded the zip, extracted the file and double clicked the executable. A log file called Depressurizer is created. But I see no GUI at all. Moreover the process keeps running in the background and I had to kill it using task manager.

Could you please look into it? The content of the log file as follows - 

```
17:20:55.932870 - INFO: Program initialized. Logging level set to Info.
17:20:55.944401 - INFO: Automatic mode not set, loading main form.
17:20:56.179526 - INFO: Loading GameDB from GameDB.xml.gz
17:20:56.516923 - INFO: GameDB XML document parsed.
17:20:57.073904 - INFO: GameDB XML processed, load complete.
```

**Comment by PaddyFloch** [comment]
Close your Steam Client, end Depressurizer's process in the task manager and start it up again, you should see it pop up after awhile.
Merry Christmas.

**Comment by iShafayet** [comment]
Thank you. That fixed it. Maybe we should show a some sort of loading screen, for the convenience of the user.

**Comment by Theo47** [comment]
Apparently i had forgotten to add a loading message for when updating data from HLTB at startup. Assuming that was the cause, this should be fixed now (6139206b2252e0c3b43cba4b075e6ad666f8b94b).
